custom_components/tuya_local/devices/LSC_SMART_Connect_RGB-CCT_Bulb

```diff
@@ -0,0 +1,106 @@
+name: LSC RGB-CCT
```

Comment:
This name should just be generic.  Quite likely other bulbs match this, as it seems to follow the older standard layout tuya bulbs use (though none exactly match it).

Reply:
Agree, i change it to LSC RGB-CCT